geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -180,6 +199,58 @@ abstract class RasterRDD[K: ClassTag] extends TileRDD[K] {
   def bands(bands: java.util.ArrayList[Int]): RasterRDD[K] =
     withRDD(rdd.mapValues { multibandTile => multibandTile.subsetBands(bands.asScala) })
 
+  def toGeoTiffRDD(
+    storageMethod: StorageMethod,
+    compression: String,
+    colorSpace: Int,
+    headTags: java.util.Map[String, String],
+    bandTags: java.util.ArrayList[java.util.Map[String, String]]
+  ): JavaRDD[Array[Byte]] = {
+    val tags =
+      if (headTags.isEmpty || bandTags.isEmpty)
+        Tags.empty
+      else
+        Tags(headTags.asScala.toMap,
+          bandTags.toArray.map(_.asInstanceOf[scala.collection.immutable.Map[String, String]]).toList)
+
+    val options = GeoTiffOptions(
+      storageMethod,
+      TileRDD.getCompression(compression),
+      colorSpace,
+      None)
+
+    toGeoTiffRDD(tags, options)
+  }
+
+  def toGeoTiffRDD(
```

Comment:
This guy can move up to `TileRDD` along with the related abstract methods since no type tie them to either `RasterRDD` or `TiledRasterRDD`